de/factorization.py

```diff
@@ -60,7 +61,58 @@ def factorizeEstimate(data, tol=1e-9, maxiter=10000):
 
 
 def cellLineFactorization(cellLine):
+    """ Import a cell line, fit the model, and return the result. """
     data, annotation = importLINCS(cellLine)
     w, eta = factorizeEstimate(data)
-
-    return w, eta, annotation
+    return w, eta, annotation[0].tolist()
+
+def cellLineComparison(cellLine1, cellLine2):
```

Comment:
Can you write a test that calls this for two cell lines?

Reply:
This function is called in test_cellLines()